### DISCUSSION TITLE
Where the heck are the user inputs????

### DISCUSSION THREAD

**Discussion by DotLYHiyou** [gleam-lang/gleam#2689] [Questions & support] [answered]
I came from python so this is so confusing!!!

**Comment by lpil** [comment]
What do you mean, sorry?

**Reply by annaxiomm** [reply, marked as answer]
I believe they mean a function to grab user input from cmd, like

```python
x = input()
```

**Reply by lamdor** [reply]
https://hexdocs.pm/gleam_erlang/gleam/erlang.html#get_line 

It is something that is implementation specific. There isn't any clear javascript analogue. 

Maybe there should be a link to the gleam_erlang package on https://gleam.run/documentation/  to help making that easier to discover? I'm not sure it'd be possible to do something on the tour since that's all generic Gleam.
